episodes/02-variables.md

```diff
@@ -221,4 +193,9 @@ var tt: real;                   // for temporary results when computing the temp
 const mindif = 0.0001: real;    // smallest difference in temperature that would be accepted before stopping
 const n = 20: int;              // the temperature at the desired position will be printed every n iterations
 ```
-{: .source}
+
+::::::::::::::::::::::::::::::::::::: keypoints
+- "A comment is preceded with `//` or surrounded by `/* and `*/`"
+- "All variables hold a certain type of data."
```

Comment:
Not sure I understand the objection. After a variable is cast in a type that is capable of representing the value, it will have a type. The point here is that all variables have a type.

Reply:
The User Guide is careful about variable initialization. It is true and important to convey that "variables are typed," but the specific statement that "all variables hold a _certain_ type" is misleading. The Chapel compiler will choose a type based on the value, if a type is not provided; but this implicitly-cast variable may be of _any_ type that can represent the value. This is not "certain," it's probably implementation-dependent.

Reply:
Ok, I reformulated the key point -- will the new version work?